CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 - Added libcudadebugger.so to nvliblist.conf to support cuda-gdb in CUDA 12+.
 
+### Bug fixes
+
```

Comment:
We don't distinguish between type of change in bug fix (x.y.z) releases, so please remove these 3 lines.